resources/shaders/accumulation.hlsl

```diff
@@ -17,11 +17,7 @@ void main(uint3 DTid : SV_DispatchThreadID)
 	float2 resolution;
 	input.GetDimensions(resolution.x, resolution.y);
 	
-	float2 uv = float2(DTid.xy) / resolution;
-	float gamma = 2.2;
-	float exposure = 1;
-
-	float3 color = input[DTid.xy].rgb / (frame_idx);
+	float3 color = input[DTid.xy].rgb / (frame_idx + 1);
```

Comment:
Why do you divide by frame_idx? Means the color would be divided by 1, then 2, then 3?

Reply:
Yes

Reply:
Hmm, I guess the var name is correct, but frame_idx is also referenced to as the current render index